packages/cacti-copm-test/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@hyperledger-cacti/cacti-copm-test",
+  "version": "2.0.0",
+  "description": "COPM testing.",
+  "keywords": [
+    "Hyperledger",
+    "Cacti",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger-cacti/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger-cacti/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger-cacti/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cacti Contributors",
+    "email": "cactus@lists.hyperledger.org",
+    "url": "https://www.hyperledger.org/use/cacti"
```

Comment:
This URL now automatically redirects to https://www.lfdecentralizedtrust.org/projects/cacti, so perhaps you can change it.

Reply:
This has been updated.